.github/workflows/test_build_wheels_linux_without_cuda.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - repository: pytorch/torchtune
-            package-name: torchtune
+          - repository: pytorch/ao
```

Comment:
Testing with `pytorch/torchtune` always fails because it requires torchao. So, I will just switch to torchao here as the test example.